plugins/kubernetes/app/models/watchers/cluster_pod_error_watcher.rb

```diff
@@ -2,34 +2,38 @@ module Watchers
   class ClusterPodErrorWatcher < BaseClusterWatcher
     include Celluloid::Notifications
 
-    def initialize(client)
-      @client = client
-      super(@client.watch_events(field_selector: 'involvedObject.kind=Pod'))
+    def initialize(cluster)
+      super(cluster.client.watch_events(field_selector: 'involvedObject.kind=Pod'))
+      @cluster = cluster
     end
 
     private
 
+    attr_reader :cluster
```

Comment:
Just thought it was cleaner to have this instead of the `@` spread across the code.
Then I was digging around it and saw some code examples with this things declared at the top after the `private` keyword, so I thought it was a common approach.
Regarding the place, I tend to declare all public methods and the the private methods, so, was not sure where to put it to be honest. :)

I'll take it out.